LICENSE.md

```diff
@@ -1 +1,123 @@
+<ol>
+  <li>In Paragraph 1, please provide the location at which a version of the CPAL License specific to the licensed Software will be made available.</li>
+  <li>"Original Code" – Please provide the title used to refer to the Software.</li>
+  <li>"Initial Developer", "Original Developer" and "Contributor" – These terms are used to refer to the individuals or entities which produced the original Software, subsequent works licensed under the CPAL License (potentially including portions of Larger Works), and Modifications, respectively.</li>
+  <ol type="a">
+    <li>If Licensor is the only attributed developer of the Software, the fields designating the "Original Developer" and "Contributor" may be left blank, and the fields designating the "Initial Developer" should refer to Licensor.</li>
+    <li>If the Software requires attribution to multiple parties involved in its development, please refer to the terms of the CPAL License Agreement to ensure proper attribution.</li>
+  </ol>
+</ol>
+
+### EXHIBIT A. Common Public Attribution License Version 1.0.
+
+"The contents of this file are subject to the Common Public Attribution License Version 1.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at **https://github.com/ChildMindInstitute/mindlogger-admin/blob/master/LICENSE.md**. The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific language governing rights and limitations under the License.
+
+The Original Code is **https://github.com/ChildMindInstitute/mindlogger-admin**.
+
+The Original Developer is the Initial Developer and is __________. If left blank, the Original Developer is the Initial Developer.
+
+The Initial Developer of the Original Code is **the MATTER Lab at the Child Mind Institute**. All portions of the code written by **the MATTER Lab at the Child Mind Institute** are Copyright (c) **2017**. All Rights Reserved.
+
+Contributor ______________________.
+
+
+### EXHIBIT B. Attribution Information.
+
+Attribution copyright notice: **Copyright (c) 2017 MATTER Lab at the Child Mind Institute**
+
+Attribution phrase: **Child Mind Institute product intended for building applications for good.**
+
+Attribution URL: **https://matter.childmind.org**
+
+Graphic image: <img width="200" src="https://github.com/ChildMindInstitute/mindlogger-admin/blob/master/CMI_logo.jpg" class="img-responsive img-centered" alt="CMI logo">
```

Comment:
This showing as a broken image – perhaps there's a better URL to link to? Or we could just remove this line (I don't particularly see the value of embedding a corporate logo in a license file).

Reply:
Makes sense, I just copied the old file and checked for repo urls, totally skipped this one 👍